### PR TITLE
Classes rename with Utils or Helper at the end

### PR DESCRIPTION
**Pull request by evasileva-epam**
It was decided with @mf-code to rename classes that are really utils with the additional end at the name "Utils" or "Helper". "Utils" for more common classes(e.g. numbers rounding), "Helper" for more rarely used(e.g. for specific page).
(*A utility class (aka helper class) is a “structure” that has only static methods and encapsulates no state.)

List of the classes from codacy issues list for https://github.com/jdi-testing/jdi-light/issues/1263 that are not renamed:
di-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/CheckListSteps.java
jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navs/ClickVerification.java
jdi-light-bootstrap-tests/src/test/java/io/github/epam/test/data/Users.java
jdi-light-html-tests/src/test/java/io/github/epam/CustomDriverTestsInit.java
jdi-light-html-tests/src/test/java/io/github/epam/html/tests/issues/issue69/JDISite69.java
jdi-light-html-tests/src/test/java/io/github/epam/html/tests/site/steps/States.java
jdi-performance/src/test/java/org/mytests/tests/states/States.java

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-examples/src/test/java/io/github/epam/test/data/ListDataUtils.java  1
- jdi-light/src/main/java/com/epam/jdi/light/driver/get/RemoteDriverUtils.java  6
- jdi-light-examples/src/test/java/io/github/epam/test/data/TableDataHelper.java  1
- jdi-light-examples/src/main/java/selenium/site/data/DefaultDataProviderHelper.java  1
- jdi-light-bdd-tests/src/main/java/io/github/com/entities/UsersUtils.java  1
- jdi-light/src/main/java/com/epam/jdi/light/actions/ActionOverrideUtils.java  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4463420&bid=15076909)

**Comment by evasileva-epam** [comment]
@RomE-HK, at https://app.codacy.com/ codacy doesn't show that any issue was fixed. But probably it's a defect of codacy as it was for NPath Complexity.

**Comment by RomE-HK** [comment]
Finally, two hours later Codacy began to show that some of issues was fixed =)
pic

**Comment by evasileva-epam** [comment]
@RomE-HK, it seems that I'm not in the list to see such things. I haven't access for "jdi-testing", only "jdi-light". It could be the reason.

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/test/data/Users.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import io.github.com.entities.User;
 
+@SuppressWarnings("PMD.ClassNamingConventions")
 public class Users {
```
Comment (RomE-HK) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/test/data/Users.java]: This class looks similar to Users(Utils) above. I think we should renamed it too

File: jdi-light-examples/src/main/java/selenium/site/data/DefaultDataProviderHelper.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
 package selenium.site.data;
 
-public class DefaultDataProvider {
+public class DefaultDataProviderHelper {
```
Comment (RomE-HK) [inline jdi-light-examples/src/main/java/selenium/site/data/DefaultDataProviderHelper.java]: Suggest using "Utils" here. I explained my point of view in review comment

File: jdi-light-examples/src/test/java/io/github/epam/test/data/TableDataHelper.java [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 import io.github.epam.entities.UserInfo;
 
-public class TableData {
+public class TableDataHelper {
```
Comment (RomE-HK) [inline jdi-light-examples/src/test/java/io/github/epam/test/data/TableDataHelper.java]: Suggest using "Utils" here. I explained my point of view in review comment

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-examples/src/test/java/io/github/epam/test/data/TableDataUtils.java  1
- jdi-light/src/main/java/com/epam/jdi/light/driver/get/RemoteDriverUtils.java  6
- jdi-light/src/main/java/com/epam/jdi/light/actions/ActionOverrideUtils.java  4
- jdi-light-examples/src/main/java/selenium/site/data/DefaultDataProviderUtils.java  1
- jdi-light-bdd-tests/src/main/java/io/github/com/entities/UsersUtils.java  1
- jdi-light-examples/src/test/java/io/github/epam/test/data/ListDataUtils.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navs/ClickVerificationUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4463420&bid=15076909)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-examples/src/test/java/io/github/epam/test/data/TableDataUtils.java  1
- jdi-light/src/main/java/com/epam/jdi/light/driver/get/RemoteDriverUtils.java  6
- jdi-light-examples/src/test/java/io/github/epam/test/data/ListDataUtils.java  1
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navs/ClickVerificationUtils.java  1
- jdi-light-examples/src/main/java/selenium/site/data/DefaultDataProviderUtils.java  1
- jdi-light-bdd-tests/src/main/java/io/github/com/entities/UsersUtils.java  1
- jdi-light/src/main/java/com/epam/jdi/light/actions/ActionOverrideUtils.java  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4463420&bid=15076909)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light/src/main/java/com/epam/jdi/light/driver/get/RemoteDriverUtils.java  6
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navs/ClickVerificationUtils.java  1
- jdi-light/src/main/java/com/epam/jdi/light/actions/ActionOverrideUtils.java  4
- jdi-light-bdd-tests/src/main/java/io/github/com/entities/UsersUtils.java  1
- jdi-light-examples/src/main/java/selenium/site/data/DefaultDataProviderUtils.java  1
- jdi-light-examples/src/test/java/io/github/epam/test/data/TableDataUtils.java  1
- jdi-light-examples/src/test/java/io/github/epam/test/data/ListDataUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4463420&bid=15076909)

**Comment by evasileva-epam** [comment]
@RomE-HK, please read comment to last commit for jdi-light-bootstrap-tests/src/test/java/io/github/epam/test/data/UsersUtils.java

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/test/data/UsersUtils.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package io.github.epam.test.data;
+
+import io.github.com.entities.UserUtils;
+
+public class UsersUtils {
+    public static UserUtils DEFAULT_USER = new UserUtils();
+}
```
Comment (RomE-HK) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/test/data/UsersUtils.java]: Empty line needed. Please, add it

Comment (evasileva-epam) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/test/data/UsersUtils.java]: Ok, I'll do it in next PR.

**Pull request merged**